### ISSUE TITLE
How to apply this model for regression

### ISSUE THREAD

**Issue by Taimoor-R** [facebookresearch/TimeSformer#121]
I was hoping you can guide me on how to use the pretrained model for regression instead of classification, for example using negative pearson loss, and each frame of the video having a unique numeric label/ground truth. So essentially the training data would be a 60 sec video broken into frames with corrsponding values/ labels for each frame.

**Comment by ashwing123** [comment]
Were you able to figure this out?
